### ISSUE TITLE
LexRuntime no longer installed as dependency of Amplify. "Can't resolve 'aws-sdk/clients/lexruntime' in @aws-amplify/interactions/lib/Providers"

### ISSUE THREAD

**Issue by jkeys-ecg-nmsu** [aws-amplify/amplify-js#2392]
**Describe the bug**
When upgrading AWS SDK for JS and Amplify to the latest version with `npm i aws-sdk@2.9.0` and  `npm i aws-amplify@latest`, I get an error that:

 `Module not found: Error: Can't resolve 'aws-sdk/clients/lexruntime' in '/.../node_modules/@aws-amplify/interactions/lib/Providers'`

**To Reproduce**
1. Install aws-sdk with `npm install --save aws-sdk@2.9.0`
2. Install Amplify with `npm install --save aws-amplify@latest`
3. Import code with `import AWS from 'aws-sdk'`

**Expected behavior**
Upgrading aws-amplify to the latest version should require that aws-sdk be upgraded to a version containing the lex runtime.

**Comment by elorzafe** [comment]
@jkeys-ecg-nmsu aws-sdk@2.9.0 is from 2 years ago, probably that doesn't have lexRuntime, aws-amplify has aws-sdk as dependency (2.329.0). I suggest to not include aws-sdk as a dependency of your app. Let me know how it goes.

**Comment by jkeys-ecg-nmsu** [comment]
@elorzafe My app is embedding a Sumerian scene that uses Amplify's version of aws-sdk, will Sumerian still have access to aws-sdk (without tree-shaking away all the services I need in Sumerian) if it's not a direct dependency?

**Comment by elorzafe** [comment]
You can import aws-sdk like this (the one that uses amplify library)

```js
import { AWS } from "@aws-amplify/core/lib/Facet";
```

**Comment by elorzafe** [comment]
@jkeys-ecg-nmsu that shouldnt be an issue, when you run `amplify init` it will add to `.gitignore` what is not required by the cli. In case the cli doesnt work, you can run `amplify init` again and that will solve the issue.

**Comment by jkeys-ecg-nmsu** [comment]
@elorzafe Ah fair enough, for whatever reason I got the impression that `amplify init` is not idempotent. I manually edited `project-config.json` when checked out on a different machine and `amplify publish` worked. It looks like that file's the only one that contains anything specific to a given machine. Is it possible to turn the absolute path for `projectPath` into a relative path? (So anyone with git access could checkout an Amplify project and do e.g. `amplify publish` without having to run `amplify init`. Or, maybe check to see if the project contains Amplify folders/files on any command besides `amplify init`, and ask to re-initialize the project if they exist?)
